src/protocols/hpr_http_router.erl

```diff
@@ -0,0 +1,88 @@
+%%%-------------------------------------------------------------------
+%%% @author jonathanruttenberg
+%%% @copyright (C) 2022, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 21. Sep 2022 12:10 PM
+%%%-------------------------------------------------------------------
+-module(hpr_http_router).
+-author("jonathanruttenberg").
+
+-include("../grpc/autogen/server/config_pb.hrl").
```

Comment:
Lets try to avoid using record definitions `hpr_route` should provide function to get all you need

Reply:
Done.